### ISSUE TITLE
Expose metrics in Prometheus

### ISSUE THREAD

**Issue by F-Legrand** [ovh/public-cloud-roadmap#608]
### User Story

As a customer 
I want the metrics of my services to be expose as Prometheus compatible
so that I could plug my service metrics and manage my own observability service.

### Acceptance Criteria

Ask us anything here in the comments below, and vote with emojis !
👍 to vote for this issue

### Discuss on Discord

Feel free to discuss with us on https://discord.com/invite/PwPqWUpN8G

**Comment by F-Legrand** [comment]
Available through API first. Configuration through Control Panel will come beginning of Q2 (FY25)

**Comment by notz** [comment]
@F-Legrand Tested with mongodb and I get some values, but they seem not be in standard mongodb exporter format. Do you also provide some dashboards?

**Comment by F-Legrand** [comment]
Hi @notz !

Many thanks for your feedback, we'll check for the format. Regarding Dashboard, if you want to display your metrics following your needs, Prometheus use Console or you can plug a Grafana service on it (that we provide).

**Comment by thild42** [comment]
The feature could now be activated through the UI for Databases services.

**Comment by brennerm** [comment]
Huge thanks for this! ❤ 

Are there any Grafana dashboards available that are compatible with the metrics exported? In my case I try to visualize the metrics of a MySQL cluster but all dashboards I could find follow the [mysqld_exporter](https://github.com/prometheus/mysqld_exporter) metrics names which don't align with the ones exposed by you guys.

**Comment by thild42** [comment]
Indeed, the exposed metrics are agnostic of the database engine and more on the system itself than the engine.

You could follow this guide from our partner to check what are the available metrics.

**Comment by brennerm** [comment]
@thild42 Which guide? 😆 

Anyway, I found [this dashboard](https://grafana.com/grafana/dashboards/16500-mysql-overview-with-telegraf/) which apparently uses the same MySQL metrics names.

**Comment by thild42** [comment]
@brennerm - damn, forgot the link. 😅

This one: https://aiven.io/docs/integrations/prometheus-system-metrics
